docs/features/shareUrl.md

```diff
@@ -0,0 +1,26 @@
+# Share Url (map state to url)
+
+Rapid decision making can be challenging for organizations. Share url feature allows for rapid decision making by allowing users to share specific layer selections and map positions.
+
+ The share url feature enables (re-)initializing a map with a given state based on url parameters.
+
+
+## Usage and expected behaviour
+The following are steps to be taken and expected behaviour when sharing a url.
+
+1. Select layers that you would wish to share. 
+![alt text](https://raw.githubusercontent.com/onaio/gisida-react/pushlayertourl/public/assets/img/layer-selection.png)
+
+    - The layers should be pushed to url based on selected map instance.
+        
+        e.g     
+http://stage-zimbabwe.onalabs.org/?map-1-layers=zim-province-pop&map-2-layers=5w-wash-safewater-province
+
+2. Select the share button to copy link to clipboard
+    ![Test Image 4](https://raw.githubusercontent.com/onaio/gisida-react/pushlayertourl/public/assets/img/share-icon.png)
```

Comment:
Ah, can we use a share icon instead of the log-out button? Will find and send one shortly!

@kahummer @kelvin-muchiri

Reply:
https://thenounproject.com/search/?q=share&i=608833

Reply:
Oh I just realized what you have is Glyphicons' share icon, so maybe we should keep it as is.

Reply:
@cKellyDesign @kahummer Just realized the Title bar icons  don't have tooltips

Reply:
oh snap, yeah we should add `title` and `alt` attributes to each.